### ISSUE TITLE
How to set vm.memory with firecracker-ctr ?

### ISSUE THREAD

**Issue by windniw** [firecracker-microvm/firecracker-containerd#497]
```bash
# start a container with command 
firecracker-ctr run --rm --tty --net-host --memory-limit 1073741824 docker.io/amd64/ubuntu c1 bash

root@microvm:/# free -m
             total       used       free     shared    buffers     cached
Mem:           112         88         23          0         17         46
```

`CreateVM` ignore any paramaters and I want to know how to enlarge my container's memory, looks like which control by `MemSizeMib` insteadof `memory-limit` 

https://github.com/firecracker-microvm/firecracker-containerd/blob/301f04bdccf375909ba273371ab4124b5def264e/runtime/service.go#L373-L377

**Comment by kzys** [comment]
Yeah. Right now `firecracker-ctr` is just `ctr`. We have that to make sure `firecracker-containerd` is having compatible `ctr`.

Providing a new CLI that can pass `firecracker-containerd` specific commands/parameters is worth to consider, I think.

**Comment by RamyAllam** [comment]
Heey everyone 👋 
Would you please advise if there is a manual way to increase the memory or other limits such as the CPU on the created microVM until the new CLI is released? Thank you,
